gatsby-node.js

```diff
@@ -11,6 +14,10 @@ exports.onCreateWebpackConfig = ({ actions: { setWebpackConfig }, stage }) => {
       },
     })
   }
+
+  if (stage === 'develop') {
```

Comment:
Could you please add inline comments so people can understand the purpose of this?

Reply:
To debug GTM events, you will need to disable the partytown or doing some tricky to remove service and web workers.

Reply:
But i will find a better way to debug

Reply:
I don't think we need this yet. This can be a future assignment. We just need to make the tradeoffs clear.

Reply:
Ok, i discovered how to debug not the GTM, but the partytown, it's just set the debug: true and, on chrome, set the all levels console.
![image](https://user-images.githubusercontent.com/15680320/147959575-3c7d5dd4-3c1c-4950-8ddc-8a81ac591553.png)

Reply:
Great! I think this would be a good thing to have in a doc (since partytown lacks any).